Source/PLCrashMacros.h

```diff
@@ -42,7 +42,10 @@
 #endif
 
 #if defined(__cplusplus)
-#  if defined(PLCRASHREPORTER_PREFIX)
+#  define NO_OTHER_MACRO_STARTS_WITH_THIS_NAME_
+#  define IS_EMPTY_(name) defined(NO_OTHER_MACRO_STARTS_WITH_THIS_NAME_ ## name)
+#  define IS_EMPTY(name) IS_EMPTY_(name)
```

Comment:
Why did you need two macros for this?

Reply:
To enable multi pass preprocessor magic 😄